src/components/app-view/app-view.jsx

```diff
@@ -138,7 +137,7 @@ class AppView extends React.Component {
                     navigation={this.props.navigation}
                 ></AppSidebar>
 
-                <Content>{this.props.content}</Content>
+                <Content isSidebarVisible={this.state.sidebarOpened}>{this.props.content}</Content>
```

Comment:
a small note to change, which didn't occur to me last time - `isSidebarVisible` and `sidebarOpened ` - can we have a consistent naming for both, please?